requirements/dist-build-constraints.txt

```diff
@@ -1,24 +1,14 @@
 #
-# This file is autogenerated by pip-compile with python 3.10
-# To update, run:
+# This file is autogenerated by pip-compile with Python 3.10
+# by the following command:
 #
 #    pip-compile --allow-unsafe --output-file=requirements/dist-build-constraints.txt --strip-extras requirements/dist-build-constraints.in
 #
-packaging==21.3
-    # via setuptools-scm
-pyparsing==3.0.6
-    # via packaging
-setuptools-scm==6.3.2
```

Comment:
This is still broken, the transitive constraints must be present for all build deps.